src/components/HomePageItems/HomePageItems.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Row, Col } from "react-bootstrap";
+import "./HomePageItems.css";
+import HomePageItemCard from "../HomePageItemCard/HomePageItemCard";
+
+function HomePageItems() {
+  const exampleItems = [
```

Comment:
This array should be coming from HomePage component via props and then used to render the items